pkg/task/backup.go

```diff
@@ -188,10 +188,9 @@ func RunBackup(c context.Context, g glue.Glue, cmdName string, cfg *BackupConfig
 		StartVersion: cfg.LastBackupTS,
 		EndVersion:   backupTS,
 		RateLimit:    cfg.RateLimit,
-		Concurrency:  cfg.Concurrency,
+		Concurrency:  defaultBackupConcurrency,
```

Comment:
If we only need to backup a few ranges in one backup task, for example backup one table without index,  we can not control the `defaultBackupConcurrency` to a proper value.

Reply:
yes that's why we also need the "Step 1" of #394 to complete the picture.

Reply:
got it